src/components/OSCALProfileCatalogInheritanceTree.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { TreeView } from "@material-ui/lab";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+import ExpandLessIcon from "@material-ui/icons/ExpandLess";
+import { Grid, IconButton, List, Paper } from "@material-ui/core";
+import ListSubheader from "@material-ui/core/ListSubheader";
+
+export default function OSCALProfileCatalogInheritanceTree(props) {
+  const tree =
+    props.validIds.length > 0 ? (
+      <Grid item className={props.classes.inheritance}>
+        <Paper>
+          <List
+            subheader={
+              <ListSubheader
+                className={props.classes.OSCALMetadataPartiesHeader}
+                component="div"
+                id="oscal-metadata-parties"
+              >
+                Profiles/Catalog Inheritance
+              </ListSubheader>
+            }
+          >
+            <TreeView
+              defaultExpandIcon={
+                <IconButton>
+                  <ExpandMoreIcon />
+                </IconButton>
+              }
+              defaultCollapseIcon={
+                <IconButton>
+                  <ExpandLessIcon />
+                </IconButton>
+              }
+              defaultExpanded={props.validIds}
```

Comment:
Just putting this out here - is the ability to actually collapse the tree necessary? If it makes things simpler to have it permanently expanded that could work as well.

Reply:
I'm not sure that it's absolutely necessary but if we can allow users to collapse the tree we should provide it as it's something, albeit small, that could improve user experience.